pyke/lightcurve.py

```diff
@@ -157,17 +160,21 @@ class KeplerLightCurveFile(object):
     path : str
         Directory path or url to a lightcurve FITS file.
     quality_bitmask : int
-        Bitmask specifying quality flags of cadences that should be ignored.
+        Bitmask specifying quality flags of cadences that should be ignored:
+            default: recommended quality mask
+            conservative: removes more flags, known to remove good data
+            hard: removes all data that has been flagged
     kwargs : dict
         Keyword arguments to be passed to astropy.io.fits.open.
     """
 
-    def __init__(self, path, quality_bitmask=KeplerQualityFlags.DEFAULT_BITMASK,
+    def __init__(self, path, quality_mask='default',quality_bitmask=None,
```

Comment:
Let's avoid having both a `quality_mask` and `quality_bitmask` argument, that's confusing.  Let's only use `quality_bitmask` and allow it to accept both an integer or a string.  (I like `bitmask` over `mask` because it prevents confusion with pixel aperture masks).

Reply:
👍